hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestMultipartObjectGet.java

```diff
@@ -170,6 +170,7 @@ private void getObjectMultipart(int partNumber, long bytes)
         REST.get(BUCKET, KEY, partNumber, null, 100, null);
     assertEquals(200, response.getStatus());
     assertEquals(bytes, response.getLength());
+    assertEquals("3", response.getHeaderString("x-amz-mp-parts-count"));
```

Comment:
We can add a similar test for asserting the `PartsCount` by calling the head object API, `REST.head(BUCKET, KEY)`;

Reply:
Has the review been addressed yet?

Reply:
Addressed all the reviews now.